README.md

```diff
@@ -12,31 +13,50 @@ pip install --upgrade pip setuptools pipenv
 pipenv install --dev
 ```
 
-## Usage
-Extract translatable text from an eQ survey with
+## Python Package Usage
+
+`eq_translations` is packaged as a python package, though it is not currently published on pypi. 
+
+To install, replace `BRANCHNAME` with an appropriate tag or branch (or master) and run#:
 
 ```
-pipenv run python -m cli.template_extractor <schema_file> <output_directory>
+pipenv install -e git+https://github.com/ONSDigital/eq-translations.git@BRANCHNAME#egg=eq_translations
 ```
-This will output the translatable text to an POT file.
 
+You can also install it locally running the following from the root directory:
 
-After the text has been translated, create a new translated survey with:
+```
+pip install .
+```
+
+### Basic library Usage
+The library exports a `eq_translations.SurveySchema` class and `eq_translations.SchemaTranslation` class. These classes can be used directly to perform translations, or there are some helper methods available in `eq_translations.entrypoints`:
+
+`extract_template(schema_path, output_directory)`
+
+`translate_schema(schema_path, translation_path, output_directory)`
+
+`handle_compare_schemas(source_schema, target_schema)`
+
+## Usage
+Extract translatable text from an eQ survey with
 
 ```
-pipenv run python -m cli.translate_survey <schema_file> <translation_path> <output_directory>
+pipenv run python -m eq_translations.cli.template_extractor <schema_file> <output_directory>
 ```
+This will output the translatable text to an POT file.
 
-To translate all surveys in a directory run:
+
+After the text has been translated, create a new translated survey with:
 
 ```
-pipenv run python -m cli.translate_all_surveys <top_level_schema_directory>
+pipenv run python -m eq_translations.cli.translate_survey <schema_file> <translation_path> <output_directory>
```

Comment:
Is the `python -m eq_translations.cli.` part not redundant now that these are registered inside console scripts?

Reply:
They're registered as entry points if you install the package, but won't be available if you don't.

I can remove if we feel this pollutes the readme, but I figured it doesn't do any harm at the moment.

Reply:
No that's a fair point. Maybe make it explicit that these could be in two states (installed vs not installed), and that when installed they will be shimmed in the PATH

Reply:
Done